### DISCUSSION TITLE
zh/guide/drivers/dropbox

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#245] [Announcements]
# zh/guide/drivers/dropbox

Dropbox官网：https://www.dropbox.com/ 获取刷新令牌 前往：https://alist.nn.ci/tool/dropbox/request.html 有两种方式，一种是直接使用AList提供的，一种是自己新建应用; 获取方式如下（推荐使用第一种^{左侧的}^）; 如果使用自己新建的应用需要填写客户端ID和`客户端秘钥; ...

https://alist.nn.ci/zh/guide/drivers/dropbox.html

**Comment by scramjetFoam** [comment]
Hello develop, 
    I used the method of creating my own new application, but after loading the directory, I received an error message 'failed get objs: failed to list objs: {}:'

**Reply by flywhale** [reply]
+1

**Reply by flywhale** [reply]
Added all the permissions I can add.

**Reply by scramjetFoam** [reply]
Can be more specific?

**Reply by flywhale** [reply]
Same message:

failed get objs: failed to list objs: {missing_scope}:missing_scope/

<img width="1015" alt="截屏2023-08-11 10 17 55" src="https://user-images.githubusercontent.com/620640/259910211-354e5f3b-3388-4037-a688-1b27df937866.png">
<img width="691" alt="截屏2023-08-11 10 19 16" src="https://user-images.githubusercontent.com/620640/259910225-750900fc-7f4d-49ea-84e4-a97712257e24.png">

**Reply by love-Liushuang** [reply]
请问好了吗？我也遇到了这样的问题。

**Reply by love-Liushuang** [reply]
重新挂载dropbox之后 现在已经好了

**Comment by love-Liushuang** [comment]
版本3.16.3添加存储时候没找到dropbox呢

**Reply by love-Liushuang** [reply]
更新到最新版本了 已经好了

**Comment by KimmyLueng** [comment]
![image](https://github.com/alist-org/docs/assets/54270610/f2acb98f-aefc-4dba-a630-3c1cbf7bb97f)
最新版还是这样提示

**Comment by love-Liushuang** [comment]
可以加一个box网盘存储吗
